CMakeLists.txt

```diff
@@ -3,4 +3,7 @@ project(pipeline_graph)
 
 set(CMAKE_CXX_STANDARD 14)
 
-add_executable(pipeline_graph  ./test/node_edge_test.cpp)
\ No newline at end of file
+include_directories(../eigen)
```

Comment:
Look at https://github.com/cb-geo/mpm/blob/develop/CMakeLists.txt#L44 for how to include Eigen and you need an additional `cmake` folder with FindEigen3.cmake https://github.com/cb-geo/mpm/tree/develop/cmake